indico/modules/events/abstracts/controllers/management.py

```diff
@@ -34,6 +35,10 @@ class RHManageAbstractsBase(RHConferenceModifBase):
     CSRF_ENABLED = True
     EVENT_FEATURE = 'abstracts'
 
+    def _checkParams(self, params):
+        RHConferenceModifBase._checkParams(self, params)
+        self.list_generator = AbstractListGenerator(event=self.event_new)
```

Comment:
Is this the best place? `RHManageAbstractsBase` will also be used for RHs dealing with specific abstracts which probably won't need this... I'd rather add a `RHAbstractListBase` that inherits from `RHManageAbstractsBase`, adds this and then acts as a base class for all abstract list RHs that need it.